src/Model/User/User.php

```diff
@@ -168,14 +168,33 @@ public function setUserType(string $userType): void
         $this->userType = $userType;
     }
 
-    public function getAdditionalProperties(): array
+    public function getCustomData(): array
     {
-        return $this->additionalProperties;
+        return $this->customData;
     }
 
-    public function setAdditionalProperties(array $additionalProperties): void
+    public function setCustomData(array $customData): void
     {
-        $this->additionalProperties = $additionalProperties;
+        $this->customData = $customData;
+    }
+
+    public function getCustomProperty(string $propertyName)
```

Comment:
Can we have anything else than `string` without breaking the user service? Yeah, right?

Reply:
Nah, the property has to be a string, the only thing we can't guarantee is the return type